net-mail/mu/mu-1.2.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools elisp-common
+
+DESCRIPTION="Set of tools to deal with Maildirs, in particular, searching and indexing"
+HOMEPAGE="http://www.djcbsoftware.nl/code/mu/"
+SRC_URI="https://github.com/djcb/mu/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="emacs zsh-completion"
+
+# net-mail/mailutils also installes /usr/bin/mu.  Block it until somebody
+# really wants both installed at the same time.
+DEPEND="
+	dev-libs/gmime:3.0
+	dev-libs/xapian
+	dev-libs/glib:2
+	emacs? ( >=virtual/emacs-24 )
+	zsh-completion? ( app-shells/gentoo-zsh-completions )
+	!net-mail/mailutils"
```

Comment:
Alphabetize dependencies grouping together unconditional depends, version conditions, and use flag.